tools/tests/annotations-v2.swagger.json

```diff
@@ -12,6 +12,9 @@
         "Supported.Annotations.SinglePartKey": {
             "type": "object",
             "properties": {
+                "__metadata": {
+                    "$ref": "#/definitions/metadata"

```

Comment:
Don’t see this in definitions: lost?

Reply:
Good find. This is strange.

I'm running `test.cmd` to update the json files. I double checked and It doesn't generate the metadata definition.
But doesn't generate any of the reusable schemas. Hmmm the `paths` property of the OAS is empty. Perhaps that's part of the issue